app/src/main/java/nerd/tuxmobil/fahrplan/congress/base/ActionBarListActivity.kt

```diff
@@ -0,0 +1,52 @@
+package nerd.tuxmobil.fahrplan.congress.base
+
+import android.R
+import android.view.View
+import android.widget.HeaderViewListAdapter
+import android.widget.ListAdapter
+import android.widget.ListView
+
+/**
+ * + * Taken from http://stackoverflow
+ * .com/questions/18403647/actionbaractivity-of-android-support-v7-appcompat-and
+ * -listactivity-in-same-act
+ * +
+ */
+abstract class ActionBarListActivity : BaseActivity() {
```

Comment:
`AlarmList` is the only class inheriting from `ActionBarListActivity`. Therefore, I think it makes more sense to move the code into `AlarmList`.

Reply:
`AlarmList.java` is one of the files you told me do not transpile. Should I also exclude `ActionBarListActivity.java` from transpiling and whoever rewrites `AlarmList.java` will copy the code over or should I migrate the Java code? In that case, I think it would be better to do this in a separate Pull request, as it doesn't really have anything to do with the transpiling.

Reply:
You can leave it out from this pull request.